src/index.ts

```diff
@@ -18,7 +18,7 @@ import { Contravariant1 } from 'fp-ts/lib/Contravariant'
  * @category routes
  * @since 0.4.0
  */
-export type QueryValues = string | Array<string> | undefined
+export type QueryValues = string | Array<string>
```

Comment:
> Why do you propose it?

I think it was just to improve correctness. Consider this example:

```ts
declare const x: Query;
Object.entries(x).map(([, queryValues]) => {
  // This will never be `undefined`, yet the type includes `undefined`.
  queryValues
});
```

Reply:
uhmmmm... I think that the `QueryValues` definition is a kind of "legacy" of Node's `querystring` package.

With the current implementation based on standard `URLSearchParams` it shouldn't happen to have an `undefined` query value.

Maybe we can add this change to a wider `fp-ts-routing` refactoring before publishing the v1.0.0